### PR TITLE
Starting page

### PR DESCRIPTION
**Pull request by ilovebycicle**
Almost excellent starting page ready to be shown for customers

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -15,17 +15,10 @@ function App() {
...
         </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        <button class="button button3">I'm too young to die!</button>
```
Comment (GreyYar) [inline src/App.js:21]: Why do we use 2 classes here?

File: src/App.js [hunk 2]
```diff
@@ -15,17 +15,10 @@ function App() {
           className={ !isShown ? "App-logo" : "App-logo-reversed" }
           alt="logo"
         />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
+        <p style={{textAlign: "center"}}>
+          <h1>Welcome to the hell marine!</h1>
```
Comment (GreyYar) [inline src/App.js:19]: Please add the transparent gray box for all the content to improve readability

File: src/App.css [hunk 3]
```diff
@@ -33,6 +32,44 @@
...
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+}
+
+.Start-button {
+  text-align: center
+}
+
+.button {
```
Comment (GreyYar) [inline src/App.css:47]: Is there a difference between lowercase class names and started from capital letter class names?

File: src/App.css [hunk 4]
```diff
@@ -33,6 +32,44 @@
   color: #61dafb;
 }
 
+.Background-logo {
+  background-color: darkred;
+  background-image: url('./Background.jpg');
+  background-repeat: no-repeat;
```
Comment (GreyYar) [inline src/App.css:38]: Do we need all those CSS rules here?

File: src/App.css [hunk 5]
```diff
@@ -33,6 +32,44 @@
...
+  background-attachment: fixed;
+  background-size: cover;
+}
+
+.Start-button {
```
Comment (GreyYar) [inline src/App.css:43]: Did we use it somewhere?

Comment (ilovebycicle) [reply src/App.css:43]: Background-size is used for proper image display. Start-button is not and will be deleted.

File: src/App.css (hunk 4 above)
Comment (ilovebycicle) [reply src/App.css:38]: Yes, we do.

File: src/App.css (hunk 3 above)
Comment (ilovebycicle) [reply src/App.css:47]: There is no difference at all. Changed lowercase class names to looklike as others for better readability.

File: src/App.css (hunk 4 above)
Comment (GreyYar) [reply src/App.css:38]: I think we could avoid using background-repeat since we use 'cover' prop for bg size

File: src/App.js (hunk 2 above)
Comment (ilovebycicle) [reply src/App.js:19]: Made another solution and resolved this problem succsessfully.

File: src/App.js (hunk 1 above)
Comment (ilovebycicle) [reply src/App.js:21]: Because if use 1 class the button behaviour will change.

Comment (GreyYar) [reply src/App.js]: nvm, solved that for you

**Pull request merged**